### PR TITLE
Correction for the Formal syntax not found issue - 01

### PR DESCRIPTION
**Pull request by ghost**
This is for the issue #18780

I added formal syntax for these properties and the source for these syntaxes is [the JSON file](https://github.com/mdn/data/blob/main/css/properties.json)) :

- (-moz-force-broken-image-icon)
- (-moz-image-region)
- (-moz-orient)
- (-moz-outline-radius)
- (-moz-outline-radius-bottomleft)
- (-moz-outline-radius-bottomright)
- (-moz-outline-radius-topleft)
- (-moz-outline-radius-topright)
- (-moz-user-focus)
- (-moz-user-input)
- (-webkit-border-before)
- (-webkit-box-reflect)
- (-webkit-mask-attachment)
- (-webkit-mask-composite)
- (-webkit-mask-position-x)
- (-webkit-mask-position-y)
- (-webkit-mask-repeat-x)
- (-webkit-mask-repeat-y)
- (-webkit-overflow-scrolling)
- (-webkit-tap-highlight-color)
- (-webkit-text-security)
- (-webkit-touch-callout)

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/-moz-image-region/index.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ The syntax is similar to the {{CSSxRef("clip")}} property. All four values are r
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+ "-moz-image-region": "<shape> | auto";
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-image-region/index.md:50]:
```suggestion
 -moz-image-region = 
   <shape> | auto"
```

(the same applies to all the others)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Please see the suggested change.

File: files/en-us/web/css/-moz-image-region/index.md (hunk 1 above)
Comment (najmiehsa) [reply files/en-us/web/css/-moz-image-region/index.md:50]: I edit this. THANK YOU for the review.

**Review by estelle — APPROVED** [review]
Looks great. Thank you.

File: files/en-us/web/css/-moz-outline-radius-bottomright/index.md [hunk 2]
```diff
@@ -32,7 +32,10 @@ The value of `-moz-outline-radius-bottomright` is either a CSS {{cssxref("length
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-moz-outline-radius-bottomright =
+ <outline-radius>
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-outline-radius-bottomright/index.md:37]:
```suggestion
  <outline-radius>
```

File: files/en-us/web/css/-moz-outline-radius-topleft/index.md [hunk 3]
```diff
@@ -32,7 +32,10 @@ The value of `-moz-outline-radius-topleft` is either a CSS [`<length>`](/en-US/d
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-moz-outline-radius-topleft =
+ <outline-radius>
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-outline-radius-topleft/index.md:37]:
```suggestion
  <outline-radius> 
```

File: files/en-us/web/css/-moz-outline-radius-topright/index.md [hunk 4]
```diff
@@ -32,7 +32,10 @@ The value of `-moz-outline-radius-topright` is either a CSS [`<length>`](/en-US/
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-moz-outline-radius-topright =
+ <outline-radius>
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-outline-radius-topright/index.md:37]:
```suggestion
  <outline-radius>
```

File: files/en-us/web/css/-moz-outline-radius/index.md [hunk 5]
```diff
@@ -68,7 +68,10 @@ One, two, three or four `<outline-radius>` values, represents one of:
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+ -moz-outline-radius =
+ <outline-radius>{1,4} [ / <outline-radius>{1,4} ]?
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-outline-radius/index.md:73]:
```suggestion
  <outline-radius>{1,4} [ / <outline-radius>{1,4} ]?
```

File: files/en-us/web/css/-moz-user-focus/index.md [hunk 6]
```diff
@@ -57,7 +57,10 @@ By setting its value to `ignore`, you can disable focusing the element, which me
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-moz-user-focus =
+ignore | normal | select-after | select-before | select-menu | select-same | select-all | none
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-user-focus/index.md:62]:
```suggestion
  ignore | normal | select-after | select-before | select-menu | select-same | select-all | none
```

File: files/en-us/web/css/-moz-user-input/index.md [hunk 7]
```diff
@@ -48,7 +48,10 @@ For elements that normally take user input, such as a {{HTMLElement("textarea")}
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-moz-user-input =
+auto | none | enabled | disabled
```
Comment (wbamberg) [inline files/en-us/web/css/-moz-user-input/index.md:53]:
```suggestion
  auto | none | enabled | disabled
```

File: files/en-us/web/css/-webkit-border-before/index.md [hunk 8]
```diff
@@ -52,7 +52,10 @@ One or more of the following, in any order:
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-border-before =
+ <'border-width'> || <'border-style'> || <color>
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-border-before/index.md:57]:
```suggestion
  <'border-width'> || <'border-style'> || <color>
```

File: files/en-us/web/css/-webkit-box-reflect/index.md [hunk 9]
```diff
@@ -53,7 +53,10 @@ The **`-webkit-box-reflect`** [CSS](/en-US/docs/Web/CSS) property lets you refle
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-webkit-box-reflect =
+ [ above | below | right | left ]? <length>? <image>?
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-box-reflect/index.md:58]:
```suggestion
  [ above | below | right | left ]? <length>? <image>?
```

File: files/en-us/web/css/-webkit-mask-attachment/index.md [hunk 10]
```diff
@@ -46,7 +46,10 @@ If a {{CSSxRef("mask-image")}} is specified, `-webkit-mask-attachment` determine
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-webkit-mask-attachment =
+ <attachment>#
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-attachment/index.md:51]:
```suggestion
  <attachment>#
```

File: files/en-us/web/css/-webkit-mask-composite/index.md [hunk 11]
```diff
@@ -72,7 +72,10 @@ The **`-webkit-mask-composite`** property specifies the manner in which multiple
 
 ## Formal syntax
 
-{{CSSSyntax}}
+```plain
+-webkit-mask-composite =
+ <composite-style>#
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-composite/index.md:77]:
```suggestion
  <composite-style>#
```

File: files/en-us/web/css/-webkit-mask-position-x/index.md [hunk 12]
```diff
@@ -58,7 +58,10 @@ The `-webkit-mask-position-x` CSS property sets the initial horizontal position
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-mask-position-x =
+ [ <length-percentage> | left | center | right ]#
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-position-x/index.md:63]:
```suggestion
  [ <length-percentage> | left | center | right ]#
```

File: files/en-us/web/css/-webkit-mask-position-y/index.md [hunk 13]
```diff
@@ -58,7 +58,10 @@ The `-webkit-mask-position-y` CSS property sets the initial vertical position of
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-mask-position-y =
+ [ <length-percentage> | top | center | bottom ]#
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-position-y/index.md:63]:
```suggestion
  [ <length-percentage> | top | center | bottom ]#
```

File: files/en-us/web/css/-webkit-mask-repeat-x/index.md [hunk 14]
```diff
@@ -51,7 +51,10 @@ The `-webkit-mask-repeat-x` property specifies whether and how a mask image is r
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-mask-repeat-x =
+ repeat | no-repeat | space | round
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-repeat-x/index.md:56]:
```suggestion
  repeat | no-repeat | space | round
```

File: files/en-us/web/css/-webkit-mask-repeat-y/index.md [hunk 15]
```diff
@@ -51,7 +51,10 @@ The `-webkit-mask-repeat-y` property sets whether and how a mask image is repeat
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-mask-repeat-y =
+ repeat | no-repeat | space | round
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-mask-repeat-y/index.md:56]:
```suggestion
  repeat | no-repeat | space | round
```

File: files/en-us/web/css/-webkit-overflow-scrolling/index.md [hunk 16]
```diff
@@ -28,7 +28,10 @@ The `-webkit-overflow-scrolling` [CSS](/en-US/docs/Web/CSS) property controls wh
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-overflow-scrolling =
+ auto | touch
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-overflow-scrolling/index.md:33]:
```suggestion
  auto | touch
```

File: files/en-us/web/css/-webkit-tap-highlight-color/index.md [hunk 17]
```diff
@@ -35,7 +35,10 @@ A {{Cssxref("&lt;color&gt;")}}.
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-tap-highlight-color =
+ <color>
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-tap-highlight-color/index.md:40]:
```suggestion
  <color>
```

File: files/en-us/web/css/-webkit-touch-callout/index.md [hunk 18]
```diff
@@ -44,7 +44,10 @@ When a target is touched and held on iOS, Safari displays a callout information
 
 ## Formal syntax
 
-{{csssyntax}}
+```plain
+-webkit-touch-callout =
+ default | none
```
Comment (wbamberg) [inline files/en-us/web/css/-webkit-touch-callout/index.md:49]:
```suggestion
  default | none
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @najmiehsa !

**Comment by najmiehsa** [comment]
Thank you for all the reviews and for merging this PR.

**Pull request merged**